### ISSUE TITLE
DisallowEqualOperators and comparing dates

### ISSUE THREAD

**Issue by Stadly** [slevomat/coding-standard#573]
Is there a recommended way to check if two `DateTime` objects are equal that does not violate the [DisallowEqualOperators](https://github.com/slevomat/coding-standard#slevomatcodingstandardcontrolstructuresdisallowequaloperators-) sniff?

```php
$a = new DateTime('2001-02-03');
$b = new DateTime('2001-02-03');
if ($a == $b) {
    // The two objects represent the same date.
}
```

**Comment by Majkl578** [comment]
You should compare e.g. `$a->format(DateTimeInterface::RFC3339_EXTENDED) === $b->format(DateTimeInterface::RFC3339_EXTENDED)`.
